File: Sorting/BubbleSort.py
def swap(arr: list, i: int, j: int) -> list:
	temp = arr[i]
	arr[i] = arr[j]
	arr[j] = temp
	return arr

def bubbleSort(arr: list) -> list:
	for i in range(len(arr)):
		for j in range(len(arr)-i-1):
			if arr[j] > arr[j+1]:
				swap(arr, j, j+1)
			print(arr)
	return arr

File: Sorting/Ruby/BubbleSort.rb
def swap(arr,i,j)
	temp = arr[i]
	arr[i] = arr[j]
	arr[j] = temp
	return arr
end # end func

def bubbleSort(arr)
	for i in (0...arr.length) do
		for j in (0...(arr.length-i-1)) do
			if arr[j] > arr[j+1]
				then swap(arr, j, j+1)
			end # end if
		end # end inner for
	end # end outer for
	return arr
end # end func